### ISSUE TITLE
create a GitHub Action that tests web tool scripts with custom docker image

### ISSUE THREAD

**Issue by cjyetman** [2DegreesInvesting/PACTA_analysis#269]
Apparently you can directly reference a docker container in a GitHub Action, which might be ideal for testing our web tools in our specially made docker image.... for example...
```
jobs:
  my_first_job:
    steps:
      - name: My first step
        uses: docker://alpine:3.8
```

That would allow us to run a test of our web tools scripts exactly how we test locally using the `2dii/r-packages` docker image and `./bin/run-web-tool`, which should be as close to possible to what will run on the web server.

_Originally posted by @cjyetman in https://github.com/2DegreesInvesting/PACTA_analysis/issues/264#issuecomment-716911643_

**Comment by cjyetman** [comment]
I guess this is a duplicate of #268?

**Comment by maurolepore** [comment]
I don't think so. The docker image does not deal with cloning repos -- which is the focus of #268

**Comment by cjyetman** [comment]
Yes, you're correct. I was working late and thinks we're getting a bit fuzzy.

**Comment by cjyetman** [comment]
trying to achieve this in #275 

happy to get any help, because I haven't figured it all out yet

**Comment by cjyetman** [comment]
think it would be ideal if our docker repo would add n image to the GitHub Container Repository and the web tool test scripts would run on that.... see https://github.com/2DegreesInvesting/docker/issues/38

**Comment by cjyetman** [comment]
closed by #444
